admin/openwisp-monitoring/Makefile

```diff
@@ -0,0 +1,119 @@
+PKG_RELEASE:=1
+PKG_VERSION=0.1.0
+
+PKG_MAINTAINER:=Federico Capoano <support@openwisp.io>
+PKG_LICENSE:=GPL3.0-or-later
+
+PKG_SOURCE_URL:=https://github.com/openwisp/openwrt-openwisp-monitoring.git
+PKG_MIRROR_HASH:=194ed584aa5a7906d378800becb831ed5403bcc5a5c4f320f9db922720c27cbf
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_DATE:=2022-05-09
+PKG_SOURCE_VERSION:=0.1.0
+PKGARCH:=all
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/openwisp-monitoring
+	TITLE:=OpenWISP Monitoring agent
+	CATEGORY:=Administration
+	SECTION:=admin
+	SUBMENU:=openwisp
+	DEPENDS:=+netjson-monitoring +openwisp-config
+	PKGARCH:=all
+	URL:=http://openwisp.org
+endef
+
+define Package/netjson-monitoring
+	TITLE:=NetJson Monitoring
+	CATEGORY:=Administration
+	SECTION:=admin
+	SUBMENU:=openwisp
+	DEPENDS:=+libubus-lua +lua-cjson +rpcd +rpcd-mod-iwinfo
+	PKGARCH:=all
+	URL:=http://openwisp.org
+endef
+
+define Build/Compile
+endef
+
+define Package/openwisp-monitoring/conffiles
+/etc/config/openwisp-monitoring
+endef
+
+define Package/netjson-monitoring/install
+	$(INSTALL_DIR) \
+		$(1)/usr/sbin \
+		$(1)/usr/libexec \
+		$(1)/usr/lib/lua/openwisp-monitoring \
+		$(1)/etc/openwisp-monitoring
+
+	$(INSTALL_BIN) \
+		$(PKG_BUILD_DIR)/openwrt-openwisp-monitoring/files/sbin/netjson-monitoring.lua \
+		$(1)/usr/libexec/netjson-monitoring
+
+	$(INSTALL_BIN) \
+		$(PKG_BUILD_DIR)/openwrt-openwisp-monitoring/files/netjson-monitoring \
+		$(1)/usr/sbin/netjson-monitoring
+
+	$(INSTALL_DATA) \
+		$(PKG_BUILD_DIR)/openwrt-openwisp-monitoring/files/lib/openwisp-monitoring/dhcp.lua \
+		$(1)/usr/lib/lua/openwisp-monitoring/dhcp.lua
+
+	$(INSTALL_DATA) \
+		$(PKG_BUILD_DIR)/openwrt-openwisp-monitoring/files/lib/openwisp-monitoring/interfaces.lua \
+		$(1)/usr/lib/lua/openwisp-monitoring/interfaces.lua
+
+	$(INSTALL_DATA) \
+		$(PKG_BUILD_DIR)/openwrt-openwisp-monitoring/files/lib/openwisp-monitoring/monitoring.lua \
+		$(1)/usr/lib/lua/openwisp-monitoring/monitoring.lua
+
+	$(INSTALL_DATA) \
+		$(PKG_BUILD_DIR)/openwrt-openwisp-monitoring/files/lib/openwisp-monitoring/utils.lua \
+		$(1)/usr/lib/lua/openwisp-monitoring/utils.lua
+
+	$(INSTALL_DATA) \
+		$(PKG_BUILD_DIR)/openwrt-openwisp-monitoring/files/lib/openwisp-monitoring/neighbors.lua \
+		$(1)/usr/lib/lua/openwisp-monitoring/neighbors.lua
+
+	$(INSTALL_DATA) \
+		$(PKG_BUILD_DIR)/openwrt-openwisp-monitoring/files/lib/openwisp-monitoring/resources.lua \
+		$(1)/usr/lib/lua/openwisp-monitoring/resources.lua
+
+	$(INSTALL_DATA) \
+		$(PKG_BUILD_DIR)/openwrt-openwisp-monitoring/files/lib/openwisp-monitoring/wifi.lua \
+		$(1)/usr/lib/lua/openwisp-monitoring/wifi.lua
+
+	$(CP) $(PKG_BUILD_DIR)/VERSION $(1)/etc/openwisp-monitoring/
+
+endef
+
+define Package/openwisp-monitoring/install
+	$(INSTALL_DIR) \
+		$(1)/usr/sbin \
+		$(1)/etc/init.d \
+		$(1)/etc/config \
+		$(1)/etc/openwisp-monitoring
+
+	$(INSTALL_BIN) \
+		$(PKG_BUILD_DIR)/openwrt-openwisp-monitoring/files/monitoring.agent \
+		$(1)/usr/sbin/openwisp-monitoring
```

Comment:
The command script in [openwisp-config](https://github.com/openwrt/packages/blob/master/admin/openwisp-config/Makefile#L54) is written with an `_` instead of `-`. Shouldn't we unify that?
Just a note

Reply:
We can update openwisp_config to be openwisp-config since it's the only script remaining which is using the underscore.